src/colorizer/Dataset.ts

```diff
@@ -350,11 +350,10 @@ export default class Dataset {
       this.loadToBuffer(FeatureDataType.U32, this.timesFile),
       this.loadToBuffer(FeatureDataType.U16, this.centroidsFile),
       this.loadToBuffer(FeatureDataType.U16, this.boundsFile),
-      this.loadFrame(0), // load first frame to set frame dimensions. Load result will be discarded
+      this.loadFrame(0), // load first frame to set frame dimensions. Load result will be discarded (see `, ,` below)
       ...featuresPromises,
     ]);
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const [outliers, tracks, times, centroids, bounds, _loaded_frame, ...featureResults] = result;
+    const [outliers, tracks, times, centroids, bounds, , ...featureResults] = result;
```

Comment:
single underscore doesn't work?

Reply:
Fixed this! I added a special exception for underscores.